### PR TITLE
Fix static img path

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

I fixed the static img problem by adding a static img path. Sadly this was not detected by the test action as the branch was created before that action was added 😅 .

Edit: Also fixed the branch name. It still used `dev` instead of `dev-refactor`

## Type of change

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] Enhancement (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Fix

## How the change has been tested

I actually tried to provoke a build fail with the new build system and although the errors get thrown, the build doesn't fail.
I need to have a second look at that. But in the meantime the PR is fine 👍 

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by felsweg-iota — APPROVED** [review]
good, looks to me

**Comment by Dr-Electron** [comment]
@felsweg-iota I fixed a bug in the cli which basically made our test action useless 😅 
I will update the cli in this PR once my change on the cli got approved later today if you don't mind.

(Luckily I tried to create a failing commit here and discovered the bug :trollface: )

**Pull request merged**
